DSE_6.9_Release_Notes.md

```diff
@@ -19,6 +19,7 @@ If you're developing applications, please refer to the [Java Driver documentatio
 
 ## 6.9.4 DSE Core
 * Updated the JDK used to build DSE to versions `8u422` and `11.0.24`. (DSP-23997)
+* Changed the MAX_HEAP_SIZE to avoid OOM errors with new Java11 Garbage Collector. The change is done in the following scrips: bin/dse, bin/dsetool, dse-db/bin/nodetool , resources/cassandra/bin/nodetool . (DSP-24431)
```

Comment:
What about?
```
* Changed the default MAX_HEAP_SIZE in `dse`, `dsetool` `nodetool`, and `nodetool` scripts to avoid OOM errors with new Java 11 Garbage Collector. (DSP-24431)
```

Reply:
I think that will make it harder to understand if it's a typo in > `nodetool`, and `nodetool` scripts < or we are referring a two different scripts